lib/pf/pfcmd/checkup.pm

```diff
@@ -1215,26 +1215,6 @@ sub cluster {
         }
     }
 
-    _check_database_host_in_cluster();
-}
-
-
-sub _check_database_host_in_cluster {
```

Comment:
Why are we removing this check? This is still a valid check since we do not want to use `localhost` as the host value in the database configuration